<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublishedVariantsWidget</class>
 <widget class="QWidget" name="PublishedVariantsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="3" column="1">
    <widget class="QLineEdit" name="f_region">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a chromosomal region, e.g. from IGV or from GSvar genes to regions dialog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>chr:start-end</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Published by:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sample:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DBSelector" name="f_sample"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Region:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="DBSelector" name="f_published"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="DBTableWidget" name="table"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="f_db"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Database:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="search_btn">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateStatus_btn">
       <property name="text">
        <string>update ClinVar submission status</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_show_replaced">
       <property name="text">
        <string>Also show replaced published variants</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Text filter:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="f_text"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Gene:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="f_gene">
     <property name="placeholderText">
      <string>gene symbol</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBSelector</class>
   <extends>QLineEdit</extends>
   <header>DBSelector.h</header>
  </customwidget>
  <customwidget>
   <class>DBTableWidget</class>
   <extends>QTableWidget</extends>
   <header>DBTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>f_sample</tabstop>
  <tabstop>f_gene</tabstop>
  <tabstop>f_region</tabstop>
  <tabstop>f_db</tabstop>
  <tabstop>f_published</tabstop>
  <tabstop>f_text</tabstop>
  <tabstop>search_btn</tabstop>
  <tabstop>table</tabstop>
  <tabstop>updateStatus_btn</tabstop>
  <tabstop>cb_show_replaced</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
